packages/react/src/stories/ic-pagination.stories.mdx

```diff
@@ -135,3 +148,62 @@ import readme from "../../../web-components/src/components/ic-pagination/readme.
     </div>
   </Story>
 </Canvas>
+
+### Playground
+
+<Canvas>
+  <Story name="Playground"
+      args={defaultArgs}
+      argTypes={{
+        pages: {
+          control: { type: "number" },
+        },
+        paginationType: {
+          control: { type: "radio" },
+          options: ["simple", "complex"],
+        },
+        appearance: {
+          control: { type: "radio" },
+          options: ["default", "dark", "light"],
+        },
+        hideCurrentPage: {
+          control: { type: "boolean" },
+        },
```

Comment:
I think boolean, text and number controls get inferred from the defaultArgs, so we should only need paginationType and appearance in argTypes